app/sidebar.py

```diff
@@ -9,29 +9,39 @@
 
 def make_sidebar(api: PtxboaAPI):
     st.sidebar.subheader("Main settings:")
-    if "include_subregions" not in st.session_state:
-        st.session_state["include_subregions"] = False
-    if st.session_state["include_subregions"]:
-        region_list = api.get_dimension("region").index
-    else:
-        region_list = (
-            api.get_dimension("region")
-            .loc[api.get_dimension("region")["subregion_code"] == ""]
-            .index
-        )
 
-    st.session_state["region"] = st.sidebar.selectbox(
+    # get list of regions that does not contain subregions:
+    region_list = (
+        api.get_dimension("region")
+        .loc[api.get_dimension("region")["subregion_code"] == ""]
+        .index
+    )

```

Comment:
I think it would be better if source regions in the list are sorted:
```suggestion
    region_list = (
        api.get_dimension("region")
        .loc[api.get_dimension("region")["subregion_code"] == ""]
        .sort_index()
        .index
    )

```